src/template_finder.py

```diff
@@ -177,6 +177,7 @@ def search(
 
         if img.shape[0] > template.shape[0] and img.shape[1] > template.shape[1]:
             res = cv2.matchTemplate(img, template, cv2.TM_CCOEFF_NORMED)
+            self.last_res = cv2.matchTemplate(img, template, cv2.TM_CCOEFF_NORMED)
```

Comment:
Why we call this twice? That's not good for runtime : ) Just do self.last_res = cv2.matchTemplate() and use self.last_res instead of res.

Reply:
Right, that is better. I decided to keep both so I dont break anyone elses use. Fixed according to your suggestion.

Reply:
Would self.last_res = res work even?